xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -232,8 +232,8 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o)
     float center_x = (float)(0.5f * o->width  + o->x) / o->source_width;
     float center_y = (float)(0.5f * o->height + o->y) / o->source_height;
 
-    m_width  = (float)o->width  / o->source_width;
-    m_height = (float)o->height / o->source_height;
+    m_width = (float)o->width;
```

Comment:
Can you use c++ casts `static_cast<float>(o->width)`? No need to change the above lines but we should be consistent going forward.